src/utils/index.ts

```diff
@@ -20,12 +20,20 @@ export const finnUtkast = (vedtakListe: Vedtak[]): Vedtak | undefined => {
 	return vedtakListe.find(v => v.vedtakStatus === 'UTKAST');
 };
 
-export const erKlarTilBeslutter = (vedtak: Vedtak): boolean => {
-	return vedtak.beslutterProsessStatus === BeslutterProsessStatus.KLAR_TIL_BESLUTTER;
+export const erBeslutterProsessStartet = (beslutterProsessStatus:  OrNothing<BeslutterProsessStatus>): boolean => {
+	return beslutterProsessStatus !== null;
```

Comment:
Det er litt farlig å sjekke med '!==', siden det vil gi feil resultat hvis beslutterProsessStatus er undefined, mens det vil gi riktig for null. Bruk heller != for null sjekker